content/en/serverless/guide/serverless_warnings.md

```diff
@@ -100,6 +100,24 @@ Attack attempts were detected targeting the serverless application.
 
 **Resolution:** Investigate the attack attempts in ASM by clicking the **Security Signals** button to determine how to respond. If immediate action is needed, you can block the attacking IP in your WAF through the [Workflows integration][11].
 
+### Under provisioned
+
+CPU utilization for this function averaged more than 80%. This means your function may benefit from additional CPU resources.
+
+**Resolution:** Consider increasing the amount of allocated memory on your Lambda function. Increasing the amount of memory scales available CPU resources. Note this may affect your AWS bill.
```

Comment:
```suggestion
**Resolution:** Consider increasing the amount of [allocated memory][12] on your Lambda function. Increasing the amount of memory scales available CPU resources. Note this may affect your AWS bill.
```

Reply:
add 

`[12]: https://docs.aws.amazon.com/lambda/latest/dg/configuration-memory.html` below